Sources/Core/AWSClientRuntime/Protocols/AWSQuery/AWSQueryError.swift

```diff
@@ -0,0 +1,37 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import protocol ClientRuntime.BaseError
+import enum ClientRuntime.BaseErrorDecodeError
+import class ClientRuntime.HttpResponse
+import class SmithyXML.Reader
+
+public struct AWSQueryError: BaseError {
```

Comment:
The "base error" for use with AWS Query. Provides values for the common error fields.